core/src/main/java/org/fao/geonet/kernel/SchemaManager.java

```diff
@@ -963,19 +980,18 @@ private void addSchema(ApplicationContext applicationContext, Path schemaDir, El
 
         for (String fname : fnames) {
             Path filePath = path.resolve("loc").resolve(defaultLang).resolve(fname);
-            if (Log.isDebugEnabled(Geonet.SCHEMA_MANAGER))
-                Log.debug(Geonet.SCHEMA_MANAGER, "Searching for " + filePath);
+            LOGGER.debug(Geonet.SCHEMA_MANAGER_MARKER, "Searching for {}", filePath);
             if (Files.exists(filePath)) {
                 Element config = new Element("xml");
                 config.setAttribute("name", schemaName);
                 config.setAttribute("base", locBase.toUri().toString());
                 config.setAttribute("file", fname);
-                if (Log.isDebugEnabled(Geonet.SCHEMA_MANAGER))
-                    Log.debug(Geonet.SCHEMA_MANAGER, "Adding XmlFile " + Xml.getString(config));
+                if (LOGGER.isDebugEnabled(Geonet.SCHEMA_MANAGER_MARKER))
```

Comment:
Can be removed `LOGGER.isDebugEnabled(Geonet.SCHEMA_MANAGER_MARKER)`?

Reply:
Nope, statement avoids call to xml string